src/components/interface/voter/Makefile

```diff
@@ -0,0 +1,5 @@
+LIB_OBJS=
+LIBS=$(LIB_OBJS:%.o=%.a)
```

Comment:
Hmmm, this should likely be part of the `Makefile.subdir`.  Perhaps I'm mis-remembering quite a bit here.